tests/foreman/ui/test_ansible.py

```diff
@@ -565,6 +565,53 @@ def test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors(
             assert f'notice Install the {package_name} package' in notice_log['permission_denied']
             assert f'Installed: rubygem-{package_name}' in notice_log['permission_denied']
 
+    def test_positive_non_admin_hostgroup_permission(
+        self, request, function_org, function_location, target_sat
+    ):
+        """Verify Hostgroup page is accessible to non-admin user.
+
+        :id: 3b8ce895-9d4c-491d-a5eb-d292748134a5
+
+        :customerscenario: true
+
+        :BZ: 2236418
+
+        :steps:
+            1. Create a user with non-admin permissions and assign "Ansible Roles Manager" and "Ansible Tower Inventory Reader"
+            to the user.
+            2. Create a hostgroup and add in same organization and location as user.
+            3. Login to Satellite WebUI with user credentials and navigate to Configure -> Hostgroups
+
+        :expectedresults: The hostgroup created should be visible to non-admin user.
+        """
+        password = settings.server.admin_password
+
+        ansible_manager_role = (
+            target_sat.api.Role().search(query={'search': 'name="Ansible Roles Manager"'})[0].id
+        )
+        tower_inventory_manager_role = (
+            target_sat.api.Role()
+            .search(query={'search': 'name="Ansible Tower Inventory Reader"'})[0]
+            .id
+        )
+        SELECTED_ROLE = [ansible_manager_role, tower_inventory_manager_role]
+        user = target_sat.api.User(
+            role=SELECTED_ROLE,
```

Comment:
This could really confuse us later as we're using SELECTED_ROLE variable in tests for ansible roles, and here using it for user and roles 
```suggestion
        user = target_sat.api.User(
            role=[ansible_manager_role, tower_inventory_manager_role],
```

Reply:
The variable definition can differ from test to test. This is something we can skip.

Reply:
I would like to keep the original approach if it is fine with you.

Reply:
but why keeping unnecessary variable here which is nowhere used later :D